test/integration/backward_compatible/topic/ReliableTopicOnClusterRestart.js

```diff
@@ -92,6 +101,13 @@ describe('ReliableTopicOnClusterRestartTest', function () {
             messageArrived = true;
         });
 
+        // wait for the message listener to be initialized
+        const runner = Object.values(topic1.runners)[0];
+        const nextFake = sandbox.replace(runner, 'next', sandbox.fake(runner.next));
+        await assertTrueEventually(async () => {
+            nextFake.called.should.be.true;
```

Comment:
see `Using fakes instead of spies` section in https://sinonjs.org/releases/v11.1.1/fakes/ for more about this usage